Sources/xcproj/PBXProjObjects+Helpers.swift

```diff
@@ -58,17 +58,17 @@ public extension PBXProj.Objects {
     ///   - toGroup: parent group
     ///   - options: additional options, default is empty set.
     /// - Returns: all new or existing groups in the path and their references.
-    public func addGroup(named groupName: String, to toGroup: PBXGroup, options: GroupAddingOptions = []) -> [(reference: String, group: PBXGroup)] {
+    public func addGroup(named groupName: String, to toGroup: PBXGroup, options: GroupAddingOptions = []) -> [ObjectReference<PBXGroup>] {
```

Comment:
Hmm, just thinking. This could either be `[String: PBXGroup]` or `[ObjectReference<PBXGroup>]`

Reply:
I think array is better here, to make it is easier to inspect all created groups in order that will match their path order. Also I think with dictionary it will be impossible to get the last created group which is likely to be used in other methods as input parameter.